BF2AutoSpectator/game/instance_manager.py

```diff
@@ -970,6 +970,10 @@ def is_scoreboard_visible(self) -> bool:
     def rotate_to_next_player():
         auto_press_key(0x2e)
 
+    @staticmethod
+    def rotate_to_previous_player():
+        auto_press_key(0x2c)
```

Comment:
I wasn't exactly sure on this value, but from my memory of the bf2 keyboard this should be right.